Scripts/RemediationScripts/Remediate-ConfigureWAFOnAzureFrontDoorCDN.ps1

```diff
@@ -383,22 +413,28 @@ function Configure-WAFConfigurationOnFrontDoorEndpoint
 
     if ($totalEndPointsWithoutWAFConfigured  -eq 0)
     {
-        Write-Host "No EndPoint(s) found with where WAF is not configured.. Exiting..." -ForegroundColor $([Constants]::MessageType.Warning)
+        Write-Host "No EndPoint Domain(s) found with where WAF is not configured.. Exiting..." -ForegroundColor $([Constants]::MessageType.Warning)
```

Comment:
NIT: remove 'with', 'where' is sufficient

Reply:
removed